cg/meta/workflow/mip_dna.py

```diff
@@ -94,3 +99,32 @@ def get_data_analysis_type(self, case_id: str) -> str:
             )
             return AnalysisType.WHOLE_GENOME_SEQUENCING
         return analysis_types.pop() if analysis_types else None
+
+    def _get_samples_sex(self, case_obj: Case, hk_version: Version) -> dict:
+        qc_metrics_file = self.get_qcmetrics_file(hk_version)
+        analysis_sexes = self.analysis_sex(qc_metrics_file)
+        samples_sex = {}
+        for link_obj in case_obj.links:
+            sample_id = link_obj.sample.internal_id
+            samples_sex[sample_id] = {
+                "pedigree": link_obj.sample.sex,
+                "analysis": analysis_sexes[sample_id],
+            }
+        return samples_sex
+
+    def analysis_sex(self, qc_metrics_file: Path) -> dict:
```

Comment:
```suggestion
    def get_analysis_sex(self, qc_metrics_file: Path) -> dict:
```
Add more details to return hint.